lib/modules/versioning/hermit/readme.md

```diff
@@ -0,0 +1,9 @@
+Hermit versioning is a mix of `semver` and `channel`.
+
+**Semver**
+
+Hermit's semver was the untangled versions comes back from package's tag. It supports major, minor, patch and also build number in the semver. An example of that is from OpenJDK `15.0.1_9`.
```

Comment:
Can you use one sentence per line please? We (try to) use semantic linefeeds[^1] in our documentation.

[^1]: https://rhodesmill.org/brandon/2012/one-sentence-per-line/